generator/internal/rust/templates/grpc-client/transport.rs.mustache

```diff
@@ -149,7 +149,7 @@ impl super::stub::{{Codec.Name}} for {{Codec.Name}} {
           (),
           {{/ReturnsEmpty}}
           {{^ReturnsEmpty}}
-          crate::{{Service.Codec.PackageModuleName}}::{{OutputType.Codec.RelativeName}},
+          crate::{{OutputType.Codec.PackageModuleName}}::{{OutputType.Codec.RelativeName}},
```

Comment:
I was expecting a change in some generated file, but there is none because the cases where there would be are suppressed. And this shows nothing else is affected. Neat.